scripts/snipe-explore.js

```
@@ -552,7 +510,16 @@ function populateFilterGroups(plotId) {
 
 // Function to determine if a column is numerical
 function isNumerical(column) {
-    const numericCount = data.filter(row => !isNaN(parseFloat(row[column])) && isFinite(row[column])).length;
+    const uniqueValues = new Set(data.map(row => row[column]));
+    if (uniqueValues.size <= 3) {
+        return false;
+    }
+
+    const numericCount = data.filter(row => {
+        const value = row[column];
+        return !isNaN(parseFloat(value)) && isFinite(value);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)